src/3_PB/AnalisiDeiRequisiti/content.typ

```diff
@@ -1054,16 +1122,24 @@ caption: "Requisiti funzionali")
 #let requisiti_qualita = (
   (
   "Obbligatorio","Il superamento di test che dimostrino il corretto funzionamento dei servizi utilizzati e delle funzionalità implementate. La copertura di test deve essere almeno dell'80% e deve essere dimostrata tramite report.","Capitolato",
-  ),
+  ), /*
   (
   "Obbligatorio",[Il sistema deve essere testato nella sua interezza tramite #glossary("test end-to-end")],"Capitolato",
-  ),
+  ),*/
   (
   "Obbligatorio", [Viene richiesta una #glossary("documentazione") sulle scelte implementative e progettuali, che dovranno essere accompagnate da motivazioni.],"Capitolato",
   ),
+  ("Obbligatorio","Viene richiesto un Manuale Utente","Capitolato"),
   (
   "Obbligatorio","La documentazione dovrà riguardare anche problemi aperti ed eventuali possibili soluzioni da approfondire in futuro.","Capitolato"
-  )
+  ),
+  ("Desiderabile","Il sistema deve essere facile da usare, basso numero di click per raggiungere le cose desiderate","Norme di progetto"),
+  ("Desiderabile","L'amministratore pubblico deve poter saper padroneggiare il sistema in breve tempo","Norme di progetto"),
+  ("Obbligatorio","La repository di github del codice sorgente deve essere accessibile a tutti","Verbale esterno"),
+  ("Obbligatorio","Devono essere rispettati tutte le norme definite nel documento _Norme di Progetto v2.0_","Norme di Progetto"),
```

Comment:
```suggestion
  ("Obbligatorio","Devono essere rispettate tutte le norme definite nel documento Norme di Progetto v2.0","Norme di Progetto"),
```

le norme relative a cosa in particolare?

Reply:
l'ho messo in generale